HACKING.md

```diff
@@ -140,4 +144,133 @@ perform some checks in the seed SNO cluster.
 
 ```shell
 skopeo copy docker://quay.io/edge-infrastructure/recert docker://${LOCAL_REGISTRY}/edge-infrastructure/recert
+````
+
+## Setup dev backup steps
+
+### minio + oadp oprator
```

Comment:
This can be easy as running this command. Perhaps provide this option as well to bring up minio in a container.
`podman run --name minio -d -p 9000:9000 -p 9001:9001 quay.io/minio/minio server /data --console-address ":9001"
`

Reply:
Added doc